showcase/types/template-registry.ts

```diff
@@ -41,6 +41,14 @@ import MockFormSuperSelectGenericContent from '../app/components/mock/components
 import MockFormSuperSelectSelectedComponentSingle from '../app/components/mock/components/form/super-select/selected-component-single';
 import MockFormSuperSelectSelectedComponentMultiple from '../app/components/mock/components/form/super-select/selected-component-multiple';
 
+// PAGE-COMPONENTS
+import CopyButtonInDropdown from '../app/controllers/page-components/copy/button/copy-button-in-dropdown';
```

Comment:
when everything is converted over to single file components this declarations would not be needed, even now if your route template and all consuming components are single file you do not need this